app/forms/two_factor_login_options_form.rb

```diff
@@ -37,6 +37,10 @@ def selection_and_configuration_id(params)
     [selection, configuration_id]
   end
 
+  def mfa_user
```

Comment:
Looks like this is unused?

Reply:
I see it's used now (or maybe I missed before). But in almost every other case we construct an instance of `MfaContext`, we call the method `mfa_context`. Maybe we should do that for consistency.

```suggestion
  def mfa_context
```

Reply:
yes, that makes sense.